migrations/20230915234055_create_user_table.sql

```diff
@@ -1,39 +1,33 @@
 -- +goose Up
 -- +goose StatementBegin
-CREATE Type PersonType AS ENUM ("LEGAL", "REAL")
+
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
-CREATE TABLE IF NOT EXISTS Persons (
-    id UUID DEFAULT uuid_generate_v4() NOT NULL,
-    fist_name VARCHAR(32) NOT NULL,
-    last_name VARCHAR(32) NOT NULL,
-    person_type PersonType NOT NULL DEFAULT PersonType.LEGAL
```

Comment:
what do you mean with LEGAL? it is an enum, ok, the other option is REAL. but the difference is not clear.

Reply:
Legal persons present as business, corps, institutes etc. REAL persons basically  human

e.g in Turkish Gerçel ve Tüzel kişiler
https://en.wikipedia.org/wiki/Legal_person